src/api/ledger/orderbook.js

```diff
@@ -77,7 +78,7 @@ function getOrderbookAsync(account, orderbook, options, callback) {
 }
 
 function getOrderbook(account: string, orderbook: Object, options={}) {
-  return utils.promisify(getOrderbookAsync.bind(this))(
+  return utils.promisify(getOrderbookAsync).call(this,
```

Comment:
Forgive me if I asked this before, I'm getting senile, but would there be any advantage to stashing away a promisified func? Or they have state or something?